rudder-lang/src/logger.rs

```diff
@@ -2,6 +2,8 @@
 // SPDX-FileCopyrightText: 2019-2020 Normation SAS
 
 use crate::Action;
+use backtrace::{ Backtrace, BacktraceSymbol };
```

Comment:
does not look like `rustfmt` was run on this

Reply:
oopsie